core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6736,6 +6766,74 @@ components:
           $ref: '#/components/schemas/ProjectRole'
           description: the role of the user in the project
       description: an element of a list of project users and their role
+    BillingProjectAdminResponse:
+      required:
+        - billingProject
+        - workspaces
+      type: object
+      properties:
+        billingProject:
+          $ref: '#/components/schemas/RawlsBillingProject'
+        workspaces:
+          type: object
+          additionalProperties:
+            type: string
+            format: uuid
+          example:
+            "workspace1": "00000000-0000-0000-0000-000000000000"
+            "workspace2": "11111111-1111-1111-1111-111111111111"
+          description: "workspaces in the billing project with their IDs"
+    RawlsBillingProject:
```

Comment:
@marctalbott it seems like (based on your UI PR) that far fewer fields are actually returned in this response. Is it just that case that theoretically this many _could_ be returned if configured for the billing project?

Reply:
Yeah, that's correct. There are a lot of possible fields which is what leads this to be so long